web_export_view/controllers/controllers.py

```diff
@@ -1,32 +1,18 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Copyright (C) 2012 Domsense srl (<http://www.domsense.com>)
-#    Copyright (C) 2012-2013:
-#        Agile Business Group sagl (<http://www.agilebg.com>)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2016 Henry Zhou (http://www.maxodoo.com)
+# Copyright 2016 Rodney (http://clearcorp.cr/)
+# Copyright 2012 Agile Business Group
+# Copyright 2012 Therp BV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
 try:
```

Comment:
AFAIK Odoo requires Python 2.x, which will always have the `json` library. This guard is unnecessary